### DISCUSSION TITLE
Problem with getting verified as a student

### DISCUSSION THREAD

**Discussion by Netherizzium** [community/community#133696] [GitHub Education]
### Hi everyone,

So basically I'm trying to upload my identity card and it has my name and everything but it doesn't have a expiration date or any sort of date at all, my other school stuff are mostly hand written (as in the info is written on the card with a pen) what do I do

**Comment by SohanKafle** [comment]
Hi [Netherizzium](https://github.com/Netherizzium),

If your identity card lacks an expiration date and other documents are handwritten, do the following steps:

**1. Upload Your Identity Card:** Take a clear photo of it.
**2. Provide Additional Proof:** Include a photo or scan of any official school documents with your name, school name, and dates, even if handwritten.
**3. Write a Brief Explanation:** Explain that in your country or institution, ID cards do not have expiration dates, and some info is handwritten.
**4. Combine and Upload:** Combine the photos/scans and the explanation into a single PDF or image file and upload it as proof of your student status.

<h2>Here is a short example of an explanation letter:</h2>

To Whom It May Concern,

I am a student at [Your School Name]. Attached is a copy of my identity card. Please note that in our institution, student identity cards do not include expiration dates. Additionally, some of our school documents contain handwritten information.

Included with this letter are scans/photos of:

1. My student identity card.
2. [Any additional document] showing my enrollment details.

I hope this provides sufficient proof of my student status.
Thank you for your understanding.

Best regards,
[Your Name]
[Your Email Address]

This should help clarify your situation and support your verification process.

**Reply by samus-aran** [reply]
Hi @SohanKafle 👋 Thanks for helping out a fellow member 👍 

> 1. Upload Your Identity Card: Take a clear photo of it.
> 2. Provide Additional Proof: Include a photo or scan of any official school documents with your name, school name, and dates, even if handwritten.
> 
> ~3. Write a Brief Explanation: Explain that in your country or institution, ID cards do not have expiration dates, and some info is handwritten.~
> 
> 4. Combine and Upload: Combine the photos/scans and the explanation into a single PDF or image file and upload it as proof of your student status.

You don't need this extra step. We understand that students don't all have ID cards, they just need to include additional evidence.

**Comment by samus-aran** [comment]
Hi @Netherizzium 👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your document does not appear to include a date demonstrating current academic affiliation.](https://github.com/orgs/community/discussions/111352#document-does-not-appear-to-include-a-date)
- [I need to translate my evidence. What can I do?](https://github.com/orgs/community/discussions/111352#translate-my-evidence)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
